tests/unit/disco/test_api.js

```diff
@@ -17,23 +17,30 @@ describe(__filename, () => {
 
   describe('getDiscoveryAddons', () => {
     it('calls the API', () => {
-      getDiscoveryAddons({ api: apiState });
+      getDiscoveryAddons({ api: apiState, platform: 'Windows' });
```

Comment:
Do we have constants for that? Windows, Darwin, etc.

Reply:
We don't, they're different from what we usually use–I think Firefox sends them.

Reply:
oh ok, fine then

Reply:
I don't even know all the values though, it would be great to document them going forward. I'll at least add a note about that...